### DISCUSSION TITLE
help needed to interpret Pr(>F)

### DISCUSSION THREAD

**Discussion by melissaszy** [vegandevs/vegan#593] [Q&A]
hello, 

I ran adonis2 and obtained this result:
```r
             Df SumOfSqs       R2      F Pr(>F)    
length        1    2.764  0.02300 7.5849  0.001 ***
species       4    6.191  0.05151 4.2467  0.001 ***
cage          2    5.807  0.04831 7.9660  0.001 ***
weight        1    2.046  0.01702 5.6129  0.001 ***
Residual    282  102.780  0.85507                  
Total       290  120.202  1.00000                  
```

I understand R2 as the amount of diversity explained by the variable (ie length explains 2.3% of the diversity). Please let me know if I am wrong.

What does F and Pr(>F) refer to? through googling, I came to the understanding that F refers to the F statistic, but I do not understand how to interpret it. Similarly, I am not sure how to interpret Pr(>F). Is it simply a p-value or does it take into account the F statistic?

thank you for your help.

**Comment by jarioksa** [comment]
_F_  is the statistic, and _Pr(>F)_ is its _P_-value as assessed by permutation tests. _R_<sup>2</sup> indeed is commonly interpreted as proportion explained. Its interpretation depends on the type of analyis (argument `by`), see https://stats.stackexchange.com/questions/617788/

**Reply by melissaszy** [reply]
hello @jarioksa 

Thank you for your reply. I have two questions.

1. Does Pr(>F) have anything to do with the F-statistic?

2. In addition, as you mentioned, R2 is the proportion of variance explained by the parameter. So, _it is not the amount of variance found within that parameter._ Is the italicized part accurate?

Sorry for the trouble. I am using R2 in my research to determine if one parameter influences the microbiota more than another. I placed both parameters in my adonis2 formula, chose by=margin to make sure to calculate marginal effects of each parameter. The parameter with the larger R2 value output from adonis2 was deemed as the parameter that influences the microbiota more. My method got questioned when I was presenting and thus I want to make sure that I understand R2 values correctly.

cheers!

**Reply by melissaszy** [reply]
in addition, is the SumOfSqs value on the right of each parameter (in my example, length SumOfSqs is 2.764 ) referring to between group sum of squares, the residual sum of squares (in my example 102.780) referring to sum of squares within groups? I understand that the within-group and between group sum of squares will not add up to the total sum of squares (in my example 120.202) because I am using by=margin.

thanks again!
